plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -71,18 +71,22 @@ So we use 'Data.SatInt', a variant of 'Data.SafeInt' that does saturating arithm
 'SatInt' is quite fast, but not quite as fast as using 'Int64' directly (I don't know why that would be, apart from maybe
 just the overflow checks), but the wrapping behaviour of 'Int64' is unacceptable..
 
-One other wrinkle is that 'SatInt' is backed by an 'Int' (i.e. a machine integer with platform-dependent
-size), rather than an 'Int64' since the primops that we need are only available for 'Int' until GHC 9.2
-or so. So on 32bit platforms, we have much less header
-
-However we mostly care about 64bit platforms, so this isn't too much of a problem. The only one where it could be a problem
-is GHCJS, which does present as a 32bit platform. However, we won't care about *performance* on GHCJS, since
-nobody will be running a node compiled to JS (and if they do, they deserve terrible performance). So:
-if we are not on a 64bit platform, then we can just fallback to the slower (but safe) 'Integer'.
+One other wrinkle is that 'SatInt' is backed by an 'Int' (i.e. a machine integer
+with platform-dependent size), rather than an 'Int64' since the primops that we
+need are only available for 'Int' until GHC 9.2 or so. So on 32bit platforms, we
+have much less header
+
+However we mostly care about 64bit platforms, so this isn't too much of a
+problem. The only one where it could be a problem is GHCJS, which does present
+as a 32bit platform. However, we won't care about *performance* on GHCJS, since
+nobody will be running a node compiled to JS (and if they do, they deserve
+terrible performance). So: if we are not on a 64bit platform, then we can just
+fallback to the slower (but safe) 'Integer'.
```

Comment:
Is your editor auto-reflowing comments or something?

Reply:
I think I'm habitually hitting Alt-Q in emacs when I see a wide comment.  I do it all the time when I'm writing my own comments and I can't stop myself.